docs/software/media.md

```diff
@@ -111,14 +128,16 @@ Also installs:
 
 - MPD (music player daemon)
 
-![myMPD web interface screenshot](../assets/images/dietpi-software-media-mympd.gif){: width="400" height="318" loading="lazy"}
+![myMPD web interface screenshot](../assets/images/dietpi-software-media-mympd.png){: width="500" height="287" loading="lazy"}
 
 === "Access to the web interface"
 
     The web interface is accessible via port **1333**:
 
     - URL: `https://<your.IP>:1333`
 
+    Alternatively, the web interface can be accessed via HTTP (no HTTPS) port 1332.
+
 === "Transfer music to DietPi"
```

Comment:
With next release, HTTP on port 1332 will be disabled, so let's not add this info anymore.
```suggestion
=== "Transfer music to DietPi"
```